selfdrive/car/honda/fingerprints.py

```diff
@@ -1307,24 +1307,34 @@
   CAR.HRV_3G: {
     (Ecu.eps, 0x18da30f1, None): [
       b'39990-3W0-A030\x00\x00',
+      b'39990-3M0-G110\x00\x00',
     ],
     (Ecu.gateway, 0x18daeff1, None): [
+      b'38897-3M0-M110\x00\x00',
       b'38897-3W1-A010\x00\x00',
     ],
     (Ecu.srs, 0x18da53f1, None): [
       b'77959-3V0-A820\x00\x00',
+      b'77959-3M0-K840\x00\x00',
     ],
     (Ecu.combinationMeter, 0x18da60f1, None): [
       b'78108-3V1-A220\x00\x00',
+      b'78109-3M0-M310\x00\x00',
+    ],
+    (Ecu.fwdRadar, 0x18DAB0F1, None): [
```

Comment:
huh, why was there no fwdRadar for this car?

Reply:
I don't see this FW for this user, only this which looks like an alt address for the radar (1 digit off on the part number prefix)

![image](https://github.com/commaai/openpilot/assets/25857203/e9509fb1-ced2-42d3-8514-2ec5c4b2cea3)

Reply:
removed and tested!